main.js

```diff
@@ -208,7 +208,7 @@ function setOption(line, option, value) {
     var toFind = "'%%" + option + "%%'";
     var pos = line.indexOf(toFind);
     if (pos != -1) {
-        return line.substring(0, pos) + ((value !== undefined) ? value : adapter.config[option] || '') + line.substring(pos + toFind.length);
+        return line.substring(0, pos) + ((value !== undefined) ? value : (adapter.config[option] == null) ? '' : adapter.config[option]) + line.substring(pos + toFind.length);

```

Comment:
Mit `|| ' '` wird ein `false` auch ersetzt (z.B. bei checkbox!)

Ich nehme an, hier soll geprüft werden ob die variable existiert?
Das geht laut http://stackoverflow.com/questions/2647867/how-to-determine-if-variable-is-undefined-or-null mit `== null`

Allerdings sind die sich da wohl auch nicht so einig.

Hier gäbe es noch ne andere Lösung mit `typeof`
http://stackoverflow.com/questions/5113374/javascript-check-if-variable-exists-is-defined-initialized